pkgs/by-name/ge/gephgui-wry/package.nix

```
@@ -0,0 +1,129 @@
+{
+  lib,
+  rustPlatform,
+  webkitgtk_4_1,
+  pkg-config,
+  buildNpmPackage,
+  makeDesktopItem,
+  fetchFromGitHub,
+  nix-update-script,
+  perl,
+  stdenv,
+  glib,
+  copyDesktopItems,
+  wrapGAppsHook4,
+}:
+let
+  version = "5.3.1";
+  src = fetchFromGitHub {
+    owner = "geph-official";
+    repo = "gephgui-pkg";
+    rev = "v${version}";
+    hash = "sha256-JKzb8Hu91SDYs3E2U2WKX3iXif8fi6rmPglhuCXWLco=";
+    fetchSubmodules = true;
+  };
+
+  gephgui = buildNpmPackage {
+    pname = "gephgui";
+    inherit version src;
+
+    sourceRoot = "${src.name}/gephgui-wry/gephgui";
+
+    npmDepsHash = "sha256-dGzmdvzKp/JHCgDf3NJb0oolgW4Y/spagzpeVpMF28w=";
+
+    installPhase = ''
+      mkdir -p $out
+      cp -aR dist/* $out/
+    '';
+  };
+
+  pac-cmd = stdenv.mkDerivation {
+    pname = "pac-cmd";
+    version = "0-unstable-2020-01-07";
+
+    src = fetchFromGitHub {
+      owner = "getlantern";
+      repo = "pac-cmd";
+      rev = "495bad48b8601385daa8205ec4db504166dff18b";
+      hash = "sha256-T4g0FR20sOxJ20H4LTmhrA2EsRe8JYXj6MB2ImkbPsY=";
+    };
+
+    nativeBuildInputs = [ pkg-config ];
+    buildInputs = [ glib ];
+
+    buildPhase = ''
+      runHook preBuild
+
+      NIX_CFLAGS_COMPILE="$(pkg-config --cflags gio-2.0) $NIX_CFLAGS_COMPILE"
+      NIX_LDFLAGS="$(pkg-config --libs gio-2.0) $NIX_LDFLAGS"
+
+      $CC -Wall -c -D LINUX -D AMD64 main.c
+      $CC -Wall -c -D LINUX -D AMD64 linux.c
+      $CC $LDFLAGS -o pac main.o linux.o
+
+      runHook postBuild
+    '';
+
+    installPhase = ''
+      runHook preInstall
+
+      install -Dm755 -t $out/bin pac
+
+      runHook postInstall
+    '';
```

```suggestion
    postPatch = "
      rm binaries/*/pac
      substituteInPlace Makefile --replace-fail 'uname -p' 'uname -m'
    ";

    nativeBuildInputs = [ pkg-config ];

    buildInputs = [ glib ];

    installPhase = ''
      runHook preInstall

      install -Dm755 -t $out/bin binaries/*/pac

      runHook postInstall
    '';
```

And please add meta.